docs/source/api/transformers4rec.data.rst

```diff
@@ -0,0 +1,53 @@
+transformers4rec.data package
```

Comment:
I don't think we want to include these package yet (because of that betterproto issue during docs build)